### ISSUE TITLE
Flickering of images

### ISSUE THREAD

**Issue by YAPPDEF** [Baseflow/flutter_cached_network_image#258]
Scenario:
* Page 1 some logos
* Page 2 large list of logos
* Detail page with bigger image (banner)

When the images are loaded the first time, it's smooth (fading).
When I switch from page 1 to page 2 (forth and back), the same images are sometimes loaded again (even if they are already loaded) but not displayed immediately and also not fading. So it takes some milliseconds and then they appear immediately. So it's "nervous"

v1.1.1

**Comment by Kavantix** [comment]
Could you try using the current git version and see if it helps
```yaml
flutter_cached_network_image:
  git: git@github.com:renefloor/flutter_cached_network_image.git
```

**Comment by ssbaval** [comment]
I get the same issue - with  Latest version and git version. When the image loads - it blinks/flashes.

**Comment by cbenhagen** [comment]
@Kavantix reverting your commit 3d71ad0 fixes this issue for me on the current git master.

**Comment by diego-lipinski-de-castro** [comment]
Problem still happening, I tried on 1.13 and on 2rc

**Comment by Kavantix** [comment]
@diego-lipinski-de-castro please refer to #289 for the current status

**Comment by MichaelMarner** [comment]
@diego-lipinski-de-castro could you please provide a video demonstrating the problem you are seeing. Please turn slow animations on in Dart Devtools so it is really clear exactly what is happening.

**Comment by diego-lipinski-de-castro** [comment]
Yes but only tomorrow, some background I can provide is that I tested on iPhone 11 Pro Max emulator, tested on physical iPhone XR, and tested with this version on the lib: https://github.com/MichaelMarner/flutter_cached_network_image/tree/stop-flicker

**Comment by diego-lipinski-de-castro** [comment]
I can inspect tomorrow and back with more details, what i can advance is that I tested on physical iPhone XR iOS 13 and iPhone 11 pro max emulator on 13 aswell, I tested on your personal stop-flick branch btw

**Comment by votruk** [comment]
Any update on this issue?

**Comment by MichaelMarner** [comment]
I believe this is fixed by #289 

You can test on my branch on the pull request. If the PR doesn't fix the issue, then try and provide a minimal example that further demonstrates the problem and I can look into it. 

My branch has fixed the problem for my use cases.

**Comment by renefloor** [comment]
If it is not fixed by that PR please reopen this issue.
